### PR TITLE
docs(UPM-5493): update BigAnimal metrics documentaion for telegraf-fluentbit changes

### PR DESCRIPTION
**Pull request by matthewhudsonedb**
## What Changed?

- Update BigAnimal metrics page to incorporate changes in the way metrics are gathered. These metrics will now be stored in the DpMetrics_CL table going forward. Old InsightMetrics gathered data is still being collected but will be switched off.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/06_metrics.mdx [hunk 1]
```diff
@@ -86,44 +86,43 @@ removed or renamed. Where feasible, an effort will be made not to change the
...
-| pod_name                     | k8s pod name                                                  |
-| hostName                     | AKS node host name                                            |
-| container.azm.ms/clusterName | AKS cluster name                                              |
+| pod                          | k8s pod name                                                  |
+| aks_cluster_name             | AKS cluster name                                              |
 
-When querying for tags, for best performance apply filters that don't
-require inspection of tags (for example, filters by metric name) before tag-based filters.
+When querying for labels, for best performance apply filters that don't
+require inspection of labels (for example, filters by metric name) before label-based filters.
```
Comment (markhamersma-edb) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/06_metrics.mdx:110]: This reads awkwardly - I understand the concept of filtering before examining the values, and I appreciate that this is the original wording with the terminology updated.  I am unsure if this adds value to the customer perspective.  Approving as this is the existing published phrasing.

**Review by markhamersma-edb — APPROVED** [review]
LGTM - left a message on a line that may benefit from clarification.  POssibly needs input from a more customer facing team?

**Review by SebastienSyd — APPROVED** [review]
LGTM

**Comment by SebastienSyd** [comment]
@drothery-edb : we would like to update the BigAnimal metrics page as the metrics are changing the destination table, and we would like to reflect that change on the customer facing page

**Review by markhamersma-edb — APPROVED** [review]
Nice adjustment for clarity.

**Review by drothery-edb — APPROVED** [review]
This looks fine to me. I assume there are no code dependencies and that you'd like me to publish these changes in the next build. If so, I'll go ahead and merge (or you can have the pleasure).

**Pull request merged**